src/main/scala/amba/axi4/Fragmenter.scala

```diff
@@ -93,16 +97,16 @@ class AXI4Fragmenter()(implicit p: Parameters) extends LazyModule
 
         // Things that cause us to degenerate to a single beat
         val fixed = a.bits.burst === AXI4Parameters.BURST_FIXED
-        val narrow = a.bits.size =/= UInt(lgBytes)
+        val narrow = a.bits.size =/= lgBytes.U
         val bad = fixed || narrow
 
         // The number of beats-1 to execute
-        val beats1 = Mux(bad, UInt(0), maxSupported1)
+        val beats1 = Mux(bad, 0.U, maxSupported1)
         val beats = OH1ToOH(beats1) // beats1 + 1
 
         val inc_addr = addr + (beats << a.bits.size) // address after adding transfer
         val wrapMask = a.bits.bytes1() // only these bits may change, if wrapping
-        val mux_addr = Wire(init = inc_addr)
+        val mux_addr = WireInit(inc_addr)
```

Comment:
`WireDefault` is preferred. I have been thinking deprecating the initialization of a `Wire`

Reply:
Change to `WireDefault` is fine. Deprecatng `WireInit` seems like a bad idea for too much existing code.